pkg/drivers/http/options.go

```diff
@@ -125,3 +129,21 @@ func WithCookies(cookies []drivers.HTTPCookie) Option {
 		}
 	}
 }
+
+func WithAllowedHTTPCode(httpCode int) Option {
+	return func(opts *Options) {
+		if opts.AllowedHTTPCodes == nil {
+			opts.AllowedHTTPCodes = make([]int, 0, 1)
+		}
+		opts.AllowedHTTPCodes = append(opts.AllowedHTTPCodes, httpCode)
+	}
+}
+
+func WithAllowedHTTPCodes(httpCodes []int) Option {
+	return func(opts *Options) {
+		if opts.AllowedHTTPCodes == nil {
```

Comment:
It’s not necessary to check slice is nil or not before append

Reply:
Thanks, fixed.